README.md

```diff
@@ -105,6 +105,15 @@ Javascript/Typescript
 const tx = ClientTransaction.make(rpc.getProtocolVersion(), instr1, instr2);
 ```
 
+### Release v3.3.0
+
+An *experimental* contract has been added to ByzCoin making it possible to use
+Ethereum contracts. See directory `bevm`.
+
+The ByzCoin client-side API version number has changed from 1 to 2. Callers
```

Comment:
About that, I forgot to open an issue - it is #2089 now. There are two proposition. The third proposition is of course to ignore it and just go on.